src/platform/forms-system/src/js/utilities/file/checkTypeAndExtensionMatches.js

```diff
@@ -89,13 +89,13 @@ export default function checkTypeAndExtensionMatches({ file, result }) {
   const match = extension && fileTypeSignatures[extension];
   const signature = match?.sig;
 
-  if (signature && match?.mime === file.type) {
+  if (extension === 'txt') return true;
```

Comment:
Should this also check that `file.type` is txt or something along those lines? What if someone were to append `.txt` onto the end of a binary file?

Reply:
Please take a look when you get a chance. Thanks!